org.upbge.UPBGE.yml

```diff
@@ -0,0 +1,137 @@
+id: org.upbge.UPBGE
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: blender
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+  - "--filesystem=host"
```

Comment:
Why do you need this?

Reply:
> `--filesystem=host`

It seems counterintuitive to have to copy over project files to a separate home directory just to work with UPBGE. This isn't a matter of necessity, it just makes more sense. Basically, the same reason Blender needs host filesystem access.

> `--share=network`

Many Blender add-ons require networking, either for automatic updates or other functionality such as using hosted APIs.

> `--device=dri`

This is an application that relies on 3D rendering and of course GPU acceleration, this one seems self-explanatory.

> `--socket=pulseaudio` 

UPBGE has a built-in video editor that has audio playback. As this is a game engine, audio playback is also a given.